models/CFConfigService.cfc

```diff
@@ -111,6 +112,15 @@ component accessors=true singleton {
 						// Strip the version number out
 						result.version = CFHomePath.mid( findresults.pos[ 2 ], findresults.len[ 2 ] );
 					} 
+					//If it is still empty, try it for commandbox installations
```

Comment:
Was this logic actually being hit for you?  In the case of a CommandBox server, this entire check _should_ be skipped since we know everything about CommandBox servers from its metadata.  Or were you using one CommandBox installation to affect config into another CommandBox installation?

Reply:
I was using one commandbox installation of CF9 to affect config into another since we dont have a "proper" install of CF9 that we could test (we did have one but it is on an unreachable server) 

I realise this is a cutting edge but in this way I can actually pass the tests

Reply:
Cool that makes sense and it seems like a scenario worth covering.